src/GuestList/GuestListMainPage.js

```diff
@@ -1,45 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
-import { Button, Container, Table } from 'react-bootstrap';
+import { Button, Container, Table, Modal } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import { renderRsvpStats, createListOfGuests } from './utils';
+import AddForm from './AddForm';
 
-const GuestListMainPage = () => (
-  <>
-    <Container fluid="md" className="header">
-      <Container>
-        <h1>Betrothed</h1>
-      </Container>
-      <Container>
-        <h1>Guest List</h1>
-      </Container>
-      <Container>
-        <Link to="/dashboard">
-          <Button>Back to Dashboard</Button>
-        </Link>
-        <Button>Add Guest</Button>
+const GuestListMainPage = () => {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Container fluid="md" className="header">
+        <Container>
+          <h1>Betrothed</h1>
+        </Container>
+        <Container>
+          <h1>Guest List</h1>
+        </Container>
```

Comment:
Still curious about containers inside of containers. Using Row would be more appropriate, functionally and semantically. Especially with single element.